content/en/tracing/trace_collection/compatibility/dotnet-core.md

```diff
@@ -49,12 +49,60 @@ The .NET Tracer supports automatic instrumentation on the following architecture
 
 | Processor architectures                   | Support level         | Package version                        |
 | ------------------------------------------|-----------------------|----------------------------------------|
-| Windows x86 (`win-x86`)                   | [GA](#support-ga)     | latest                                 |
 | Windows x64 (`win-x64`)                   | [GA](#support-ga)     | latest                                 |
+| Windows x86 (`win-x86`)                   | [EOL](#support-eol)   | < 3.0.0 (e.g. 2.48.0)                  |
 | Linux x64 (`linux-x64`)                   | [GA](#support-ga)     | latest                                 |
 | Alpine Linux x64 (`linux-musl-x64`)       | [GA](#support-ga)     | latest                                 |
 | Linux ARM64 (`linux-arm64`)               | [GA](#support-ga)     | .NET 5+ only, added in version 1.27.0  |
 
+Note that running x86 applications on Windows x64 is supported.
```

Comment:
I suggest some changed wording so we don't confuse people with x86 in multiple places, what do you think?

```suggestion
Note that running 32-bit applications on Windows x64 is supported.
```